src/mono/msbuild/apple/data/ProxyProjectForAOTOnHelix.proj

```diff
@@ -46,10 +52,11 @@
   </ItemGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)$(MSBuildThisFileName).props" />
+  <Import Condition="'$(EnableAggressiveTrimming)' == 'true' and '$(UseNativeAotRuntime)' != 'true'" Project="$(ILLinkPropsPath)" />
 
   <Target Name="BundleTestAppleApp" DependsOnTargets="AppleBuild" />
 
-  <Target Name="_PrepareForAppleBuildAppOnHelix" DependsOnTargets="_PublishRuntimePack">
+  <Target Name="_PrepareForAppleBuildAppOnHelix">
```

Comment:
Out of curiosity, does moving `_PublishRuntimePack` from this target's DependsOnTargets up to `AppleBuildDependsOn` change anything functionally?

If this target is the primary reason to have _PublishRuntimePack, I think it would be better to keep that dependency.

Reply:
It shouldn't change the functionality. The primary reason for moving it to the `AppleBuildDependsOn` is to have a clear list of targets executed.

Reply:
Do you think it should be reverted/added as a `DependsOnTargets` in `_PrepareForAppleBuildAppOnHelix`?